index.html

```diff
@@ -1177,6 +1179,13 @@
             sync (unless <a>media mirroring</a> is used).
           </div>
           <div class="note">
+            The user agent is encouraged to pass locale information to the
```

Comment:
Nit: "locale and text direction information"?  Usually the latter is implied by the former, but text direction is called out in the other note.

Reply:
Done.